packages/snap/package.json

```diff
@@ -58,13 +58,17 @@
   "devDependencies": {
     "@metamask/snap-types": "^0.18.1",
     "@metamask/snaps-cli": "^0.18.1",
+    "@types/bn.js": "^5",
     "@types/chai": "^4.2.10",
+    "@types/chai-as-promised": "^7.1.5",
     "@types/mocha": "^7.0.2",
     "@types/sinon": "^7.5.2",
     "@types/sinon-chai": "^3.2.3",
     "@typescript-eslint/eslint-plugin": "^5.27.1",
     "@typescript-eslint/parser": "^5.27.1",
+    "bn.js": "^5.2.1",
```

Comment:
why do you need bn.js?

Reply:
`near-api-js` still uses `bn.js` for big numbers
https://github.com/near/near-api-js/blob/master/packages/near-api-js/src/transaction.ts

Reply:
but where are you using it in code?